apps/full-site-editing/tsconfig.json

```diff
@@ -2,6 +2,37 @@
 	"extends": "@automattic/calypso-build/tsconfig",
 	"exclude": [ "node_modules" ],
 	"compilerOptions": {
-		"allowJs": true
+		"target": "esnext",
```

Comment:
are these rules not part of `"@automattic/calypso-build/tsconfig"`? seems like there are a lot of ways to have different TS configs in different packages or apps

Reply:
Yeah… improving TS setup across Calypso is a project in my backlog I haven't gotten to.

I'd like to deprecate, remove, and stop extending the tsconfig in calypso-build. I don't think there's much value there and the tsconfig should just be done per-project. There should be a base one specific to the Calypso monorepo that is extended for sub-projects.

Much of what's in here doesn't matter all that much. I believe it's only used by tsserver (editor integrations) and as long as it's providing good information, that's all we can ask 🙂 

There's almost no TypeScript in FSE, but I'm working on that 😁 Even then, we'll continue to use `calypso-build` for the foreseeable future which just strips out TypeScript constructs.

I'll remove the `extends` above.